modules/lang-painless/src/main/resources/org/elasticsearch/painless/org.elasticsearch.script.ingest.txt

```diff
@@ -35,7 +35,7 @@ class org.elasticsearch.script.Metadata {
     String getVersionType()
     void setVersionType(String)
 
-    ZonedDateTime getTimestamp()
```

Comment:
We need to provide backcompat for existing scripts. It could be done through an augmentation so it is isolated to just scripts.

Reply:
Is that the case for Metadata?  It's in 8.4 but not documented or supported.

Reply:
Well, 8.4 has just been branched. So I guess this could change, but it will need to go back to that branch (it should be fine, since it was literally just branched today).